extensions/test-src/org/pentaho/platform/osgi/KarafBootTest.java

```diff
@@ -124,7 +123,13 @@ public void testBuildExtraKettleEtc() throws Exception {
 
   @Test
   public void testConfigureSystemProperties() throws Exception {
-
+    System.setProperty( "karaf.home", "karaf.home" );
+    String solutionRootPath =  "solutionRootPath";
```

Comment:
I think so. If we're treating karaf.data differently, we should validate its value is the expected